src/sidebar/components/UserMenu.tsx

```diff
@@ -45,17 +45,23 @@ function UserMenu({ frameSync, onLogout, settings }: UserMenuProps) {
     !isThirdParty || serviceSupports('onProfileRequestProvided');
   const isLogoutEnabled =
     !isThirdParty || serviceSupports('onLogoutRequestProvided');
+  const isProfileEnabled =
+    /* isThirdParty || */ store.isFeatureEnabled('client_new_profile');
```

Comment:
I was wondering if we want this item to depend on the fact that the user belongs to a third party.

Ideally I would not bind this option to that condition, and just make it depend on the feature flag, however, I don't know if we can make it show only to LMS users (for now) only based on that. If we can't, we might need to uncomment `/* isThirdParty || */` for now.